### PR TITLE
udf-broadcast-variable-concurrency-fix

### PR DESCRIPTION
**Pull request by bolcman**
adding additional  FileShare.Read flag.

### CODE REVIEW

File: src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ internal BroadcastVariables Process(Stream stream)
                     else
                     {
                         string path = SerDe.ReadString(stream);
-                        using FileStream fStream = File.Open(path, FileMode.Open, FileAccess.Read);
+                        using FileStream fStream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.Read);
```
Comment (suhsteve) [inline src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs:57]: Can you break this line apart and get it < 100 char?  There is a style guide [here](https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md)

Comment (bolcman) [reply src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs]: yes, sorry about that , just pushed the formatted line.

**Comment by suhsteve** [comment]
Hmm, would it be too much to ask for a test to cover this scenario ?

**Comment by imback82** [comment]
Yea it would be non-trivial with the current set up. We can tackle this as a follow up PR if needed.

**Comment by bolcman** [comment]
> ea it would be non-trivial with the current set up. We can tackle this as a follow up PR if needed.

**Comment by imback82** [comment]
@bolcman why is this closed? This PR is not merged yet.

**Pull request merged**